ceopardy/controller.py

```diff
@@ -16,13 +16,39 @@
 # GNU General Public License for more details.
 #
 from ceopardy.game import Game, GameBoard
+#import json
+
 
 class Controller():
+    def __init__(self):
+        self.g = Game()
+        self.gb = GameBoard()
+        # Merge various constants from model
+        self.g.config.update(self.gb.config)
+
+    def get_config(self):
+        return self.g.config
+
+    def get_nb_teams(self):
+        return self.g.config["NB_TEAMS"]
+
+    def get_question(self, column, row):
```

Comment:
Again, not sure about columns and rows... Maybe it is the right abstraction. We'll see.